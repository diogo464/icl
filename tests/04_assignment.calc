{
    let x = 5;
    let y: int = 10;
    let z: str = "Hello";
    println x + y;
    println z;
};;