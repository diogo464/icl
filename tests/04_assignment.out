15.0
Hello
